.npmrc

```diff
@@ -1,2 +1,2 @@
 auto-install-peers = true
-hoist = false
+hoist-pattern[]=vfile-message
```

Comment:
`rehype-katex` was added to `nextra` in 2.0.2 to support LaTeX which brings in `hast-util-from-html` which has a dependency issue. 

I sent them a PR here: https://github.com/syntax-tree/hast-util-from-html/pull/4, but need to patch it on our end for now.